processing/src/main/java/io/druid/query/filter/DoubleValueMatcherColumnSelectorStrategy.java

```diff
@@ -31,7 +31,7 @@
   @Override
   public ValueMatcher makeValueMatcher(final BaseDoubleColumnValueSelector selector, final String value)
   {
-    final Double matchVal = DimensionHandlerUtils.convertObjectToDouble(value);
+    final Double matchVal = DimensionHandlerUtils.convertObjectToDouble(value, false);
```

Comment:
nit: Probably better to add a method `DimensionHandlerUtils.convertObjectToDouble(value)` which internally calls `DimensionHandlerUtils.convertObjectToDouble(value, false)` for convenience.

Reply:
Added a method without the parseException boolean